pbf/tests.py

```diff
@@ -278,3 +258,58 @@ def test_fast_blitz(self):
 
     def test_slow_blitz(self):
         self.assert_cost(['Call to Vigilance'], 2, scenario='Blitz')
+
+class TestElements(TestCase):
+    def setup_game(self, card_names):
+        game = Game()
+        game.save()
+        player = GamePlayer(game=game, spirit=Spirit.objects.get(name='River'))
+        player.save()
+        cards = [Card.objects.get(name=name) for name in card_names]
+        player.play.set(cards)
+
+        return player
+
+    def assert_elements(self, player, expected_elements):
+        if (len(expected_elements) == 0):
+            self.assertEqual(len(player.elements), 0)
```

Comment:
maybe it's best to always check that the two `len` are equal! Consider a case where someone has broken the element calculation code on the player model side, so that `player.elements` is always empty e.g.:

```python
    def elements(self):
        return Counter()
```

In this case, `for e in player.elements.keys()` checks nothing, and all tests would pass, even if some elements are expected!

Reply:
I'll take care of this since I'm ready to merge it